lib/install/validate/validator.go

```diff
@@ -309,7 +310,18 @@ func (v *Validator) Validate(ctx context.Context, input *data.Data) (*config.Vir
 	v.compute(op, input, conf)
 	v.storage(op, input, conf)
 	v.network(op, input, conf)
+	// FIXME ATC DEBT setting this value needs to be moved to Dispatcher
+	// https://github.com/vmware/vic/issues/6803
+	ok := v.CheckPersistNetworkBacking(op, true)
+	if !ok {
+		err := v.ConfigureVCenter(op)
```

Comment:
should this be:
```
err := v.ConfigureVCenter(op)
v.NoteIssue(err)
```
That would trigger a failure in the case that the update fails: https://vmware.slack.com/archives/D0G98MP88/p1511121034000100

Reply:
that fails to set then we check again on 324 with quiet=false which will note issue here
https://github.com/vmware/vic/pull/6815/files/c333627948db8ef82ed9b895ab2502d95ecff37c#diff-1e16518ca9e1f0254fdf02fd7b7bd502R591

Reply:
ok - non-obvious for me, but we'll be moving this out of validator soon enough anyway.
Can you open a follow up issue for the move to Dispatcher and then this can be merged.

Reply:
👍 https://github.com/vmware/vic/issues/6803